mungegithub/run.sh

```diff
@@ -16,7 +16,7 @@
 
 while true; do
     /mungegithub \
-      --pr-mungers=blunderbuss,lgtm-after-commit,remove-needs-rebase \
```

Comment:
silly phone auto-complete...

We weren't searching all of the events, so we didn't find an LGTM when it existed and we would spuriously remove LGTMs...